src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,39 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    userId,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {user && (
+        <UserInfo key={userId} user={user} />
```

Comment:
you don't need key here, you're not mapping elements
```suggestion
        <UserInfo user={user} />
```